packages/api/amplify_api/lib/src/graphql/ws/web_socket_connection.dart

```diff
@@ -191,6 +196,7 @@ class WebSocketConnection implements Closeable {
   void _cancel(String subscriptionId) {
     _logger.info('Attempting to cancel Operation $subscriptionId');
     send(WebSocketStopMessage(id: subscriptionId));
+    // ignore: todo
```

Comment:
I don't believe the TODO should cause the lint check to fail. Not sure if that was why you added this or not.

Reply:
you are right, removed this. I must have a workspace setting for this or something bc vscode was complaining but not analyzer.